KOGITO-5564 - Release notes for breaking config changes on postgresql… (#3812)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/chap-kogito-release-notes.adoc

The following sections describe some of the new features or enhancements in {PRODUCT} {PRODUCT_VERSION}.

=== {PRODUCT} runtimes

> ==== Breaking change on {PRODUCT} Postgresql addon for Quarkus 

Reviewer: ```suggestion
==== Breaking change on {PRODUCT} PostgreSQL add-on for Quarkus 
```

> The usage of `kogito.persistence.postgresql.connection.uri` has been discontinued in this release when using Postgresql addon for Quarkus. The same configuration can now be achieved by using the https://quarkus.io/guides/reactive-sql-clients#reactive-datasource[Quarkus reactive data source].

Reviewer: ```suggestion
The usage of `kogito.persistence.postgresql.connection.uri` is discontinued in 1.9.0 when using PostgreSQL add-on for Quarkus. You can use the same configuration using https://quarkus.io/guides/reactive-sql-clients#reactive-datasource[Quarkus reactive datasource].
```

> .Example on how to configure the data source in the application.properties file:

Reviewer: ```suggestion
.Example of how to configure the datasource in the application.properties file:
```
